bundles/org.openhab.binding.luftdateninfo/README.md

```diff
@@ -76,35 +76,35 @@ Perform the following steps to get the appropriate Sensor ID
 luftdaten.things
 
 ```java
-Thing luftdateninfo:particulate:pm_sensor   "PM Sensor"         [ ipAddress=192.168.178.50 ]
-Thing luftdateninfo:conditions:cond_sensor  "Condition Sensor"  [ sensorid=28843 ]
-Thing luftdateninfo:noise:noise_sensor      "Noise Sensor"      [ sensorid=39745 ]
+Thing sensorcommunity:particulate:pm_sensor   "PM Sensor"         [ ipAddress=192.168.178.50 ]
+Thing sensorcommunity:conditions:cond_sensor  "Condition Sensor"  [ sensorid=28843 ]
+Thing sensorcommunity:noise:noise_sensor      "Noise Sensor"      [ sensorid=39745 ]
```

Comment:
This example cannot be changed without changing the underlying naming in code, otherwise it won't work.